src/cell_lists/full_grid.jl

```diff
@@ -156,15 +156,15 @@ function each_cell_index(cell_list::FullGridCellList{Nothing})
     error("`search_radius` is not defined for this cell list")
 end
 
-@inline function cell_index(cell_list::FullGridCellList, cell::Tuple)
+@propagate_inbounds function cell_index(cell_list::FullGridCellList, cell::Tuple)
     (; linear_indices) = cell_list
 
     return linear_indices[cell...]
 end
 
 @inline cell_index(::FullGridCellList, cell::Integer) = cell
 
-@inline function Base.getindex(cell_list::FullGridCellList, cell)
+@propagate_inbounds function Base.getindex(cell_list::FullGridCellList, cell)
```

Comment:
Is it intended to remove `@inline`? In `util.jl` with `@inbounds`, you kept the `@inline` macro.

Reply:
`@propagate_inbounds` implicates `@inline`. In `util.jl` I call a function, which is different.